a3p-integration/proposals/a:upgrade-next/jsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "compilerOptions": {
+    "target": "esnext",
+    "module": "esnext",
+    "moduleResolution": "node",
+    "allowJs": true,
```

Comment:
please use `tsconfig.json` so `yarn tsc` doesn't require the `-p jsconfig.json` option. You already have `allowJs: true` and I don't think it will need node_modules search depth because the dependencies have `.d.ts` typedefs

Reply:
There is no tsc in this package. This is literally to help with code editors

Reply:
I don't get your point. Code editors find `tsconfig` just as well as `jsconfig`. And if someone wants to typecheck from the CLI they're going to type `yarn tsc`, which won't find this config file.

What you said suggested you didn't think the change was worthwhile, but I see you pushed another commit without changing the filename and removed "allowJs", so I take it you have some objection to the tsconfig filename. I'm curious what it is.